experimental/InvariantTheory/src/TorusInvariantsFast.jl

```diff
@@ -206,3 +370,42 @@ function torus_invariants_fast(W::Vector{Vector{ZZRingElem}}, R::MPolyRing)
         end
     end
 end
+
+#####################Invariant rings as affine algebras
+
+@doc raw"""
+    invariant_ring_as_affine_algebra(RT::TorGrpInvRing)
+
+Return the invariant ring `RT` as an affine algebra.
+
+That is, compute the algebra syzygies among the fundamental invariants of `RT`.
+
+# Examples
+```jldoctest
+julia> T = torus_group(QQ,2);
+
+julia> r = representation_from_weights(T, [-1 1; -1 1; 2 -2; 0 -1]);
+
+julia> RT = invariant_ring(r);
+
+julia> fundamental_invariants(RT)
+3-element Vector{MPolyDecRingElem}:
+ X[1]^2*X[3]
+ X[1]*X[2]*X[3]
+ X[2]^2*X[3]
+
+julia> invariant_ring_as_affine_algebra_(RT)
+(Quotient of multivariate polynomial ring by ideal (-t[1]*t[3] + t[2]^2), Hom: quotient of multivariate polynomial ring -> graded multivariate polynomial ring)
+```
+"""
+function invariant_ring_as_affine_algebra(R::TorGrpInvRing)
```

Comment:
I think in the "usual spite" of naming functions (and the last bullet point in [Naming conventions](https://docs.oscar-system.org/dev/DeveloperDocumentation/styleguide/#Naming-conventions)), this function should rather be named `affine_algebra(R::TorGrpInvRing)`, as the first part is already clear from the type.